### ISSUE TITLE
why the loss is negative when fine-tuning your trained model on my own datasets?

### ISSUE THREAD

**Issue by tzczsq** [facebook/C3D#107]
Hi, I have a problem.
   When fine-tuning your trained model on my own datasets, some iteration produces the negative loss value ( less than zero). Normally, the loss should be more than zero, rather than less than zero? Why?

**Comment by dutran** [comment]
@tzczsq do you have any log file (and net .prototxt file). Are you sure that the class labels are trickly between 0..(n-1), where n is the number of classes.

**Comment by tzczsq** [comment]
Hi, dutran, the class labels on our datasets are 1,2,.....,n, rather than 0,1, n-1? Is it right?

**Comment by dutran** [comment]
That's not right as we are in C land (not MATLAB).

**Comment by dutran** [comment]
@tzczsq have you got your problem solved?
